backend/app/Http/Controllers/QuizController.php

```diff
@@ -23,4 +22,12 @@ public function store(Request $request)
 
         return Quiz::create($request->all());
     }
+
+    public function update(Request $request, $id)
+    {
+        $quiz = Quiz::find($id);
+        $quiz->update($request->all());
+
+        return $quiz;
```

Comment:
Have you tried testing this one, that returns the updated quiz?

Reply:
Yes sir. it returns the updated quiz